stubs/docker/docker/constants.pyi

```diff
@@ -0,0 +1,21 @@
+from collections.abc import Mapping, Sequence
+
+DEFAULT_DOCKER_API_VERSION: str
+MINIMUM_DOCKER_API_VERSION: str
+DEFAULT_TIMEOUT_SECONDS: int
+STREAM_HEADER_SIZE_BYTES: int
+CONTAINER_LIMITS_KEYS: Sequence[str]
+DEFAULT_HTTP_HOST: str
+DEFAULT_UNIX_SOCKET: str
+DEFAULT_NPIPE: str
+BYTE_UNITS: Mapping[str, int]
+INSECURE_REGISTRY_DEPRECATION_WARNING: str
+IS_WINDOWS_PLATFORM: bool
+WINDOWS_LONGPATH_PREFIX: str
+DEFAULT_USER_AGENT: str
+DEFAULT_NUM_POOLS: int
+DEFAULT_NUM_POOLS_SSH: int
+DEFAULT_MAX_POOL_SIZE: int
+DEFAULT_DATA_CHUNK_SIZE: int
+DEFAULT_SWARM_ADDR_POOL: Sequence[str]
+DEFAULT_SWARM_SUBNET_SIZE: int
```

Comment:
Optional: Considering this module is called "constants", would it make sense to annotate these using `Final`?

Reply:
done